packages/orchestration/test/exos/local-chain-account-kit.test.ts

```diff
@@ -77,6 +77,57 @@ test('deposit, withdraw', async t => {
   );
 });
 
+test('delegate, undelegate', async t => {
+  const { bootstrap, brands, utils } = await commonSetup(t);
+
+  const { bld } = brands;
+
+  const { timer, localchain, marshaller, rootZone, storage } = bootstrap;
+
+  t.log('chainInfo mocked via `prepareMockChainInfo` until #8879');
+  const agoricChainInfo = prepareMockChainInfo(rootZone.subZone('chainInfo'));
+
+  t.log('exo setup - prepareLocalChainAccountKit');
+  const { makeRecorderKit } = prepareRecorderKitMakers(
+    rootZone.mapStore('recorder'),
+    marshaller,
+  );
+  const makeLocalChainAccountKit = prepareLocalChainAccountKit(
+    rootZone,
+    makeRecorderKit,
+    // @ts-expect-error mocked zcf. use `stake-bld.contract.test.ts` to test LCA with offer
+    Far('MockZCF', {}),
+    timer,
+    timer.getTimerBrand(),
+    agoricChainInfo,
+  );
+
+  t.log('request account from vat-localchain');
+  const lca = await E(localchain).makeAccount();
+  const address = await E(lca).getAddress();
+
+  t.log('make a LocalChainAccountKit');
+  const { holder: account } = makeLocalChainAccountKit({
+    account: lca,
+    address,
+    storageNode: storage.rootNode.makeChildNode('lcaKit'),
+  });
+
+  t.regex(await E(account).getAddress(), /agoric1/);
+
+  await E(account).deposit(await utils.pourPayment(bld.units(100)));
+
+  const validatorAddress = 'agoric1validator1';
+
+  // Because the bridge is fake,
+  // 1. these succeed even if funds aren't available
+  // 2. there are no return values
+  // 3. there are no side-effects such as assets being locked
+  await E(account).delegate(validatorAddress, bld.units(999));
+  // TODO get the timer to fire so that this promise resolves
```

Comment:
When do we address this TODO?

For `undelegate` in `stakingAccountKit`, I mocked the lca to simulate responses to transactions:
https://github.com/Agoric/agoric-sdk/blob/dc-unk-chain-misc/packages/orchestration/test/mockAccount.ts

This arrangement allowed me to test the timer stuff:

https://github.com/Agoric/agoric-sdk/blob/a5623dbf2f0975c9761e86ddca3b39fde09eb121/packages/orchestration/test/staking-ops.test.ts#L237-L251

Reply:
hm... looks like `fakeBridge` handles the simulated tx response; maybe that's orthogonal to the timer stuff.